src/sign/hashedTypes.json

```diff
@@ -64,5 +64,15 @@
   "d56782e3b50ac86c25ae292923da8c367e3c9e8e7ea9d8baa435051fe2f430fa": "proposal",
   "df10a7eeabe19301d6018be8b6c5d13231320d7ece64d021043fa172b64f3796": "update-proposal",
   "beda1f464a6112f9ed6335c4614e32a97f0e18ef4ac10b4b1c8239c475f2d8e8": "proposal",
-  "ff74674f39ca59b60056ecddaada0cb513c4729e634e99cb778f53ee404ac806": "update-proposal"
+  "ff74674f39ca59b60056ecddaada0cb513c4729e634e99cb778f53ee404ac806": "update-proposal",
+  "e037b37311c75c995f732b57cd456bdc685e5eac29668698be34388cafb14097": "proposal",
+  "9c66485d2c9af4010e89b0352e3f78ce2aa96661c531b1bdbd7042388867e830": "update-proposal",
+  "dd19ee4357e5bc813529e1b537b77ccb767135701f0223434f3b53f1ac03dcc6": "delete-proposal",
+  "817265460009cfdbde0752fcc7ef629ae7e05a6e8b6cbffab8a7cbf6cd19e87e": "delete-proposal",
```

Comment:
Two new delete-proposal 🤔 i think only one should be added for current change

same for vote vote-string and vote-array, there are two added